hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/mor/MergeOnReadTableState.java

```diff
@@ -82,6 +85,10 @@ public int getOperationPos() {
     return operationPos;
   }
 
+  public String getMergeClass() {
+    return mergeClass;
+  }
```

Comment:
No need to pass the string in this POJO, IMO.

Reply:
that`s right,ths